i18n/en.pot

```diff
@@ -532,6 +532,9 @@ msgstr "Edit"
 msgid "Share"
 msgstr "Share"
 
+msgid "Saving..."
+msgstr "Saving..."
```

Comment:
I just realized we're adding translations only to the en.pot file, not the .po files. Not sure how upstream with take it.